src/js/components/projects/detail.tsx

```diff
@@ -126,6 +139,15 @@ const ProjectDetail = (props: RouteComponentProps) => {
           // Fetching tasks from API
           <SpinnerWrapper />
         )}
+        {readyToSubmit && (
+          <SubmitModal
+            taskId={project.id}
```

Comment:
```suggestion
            instanceId={project.id}
```

Reply:
Also missing `instanceType` -- check all of these props against the expected ones in `submit.tsx`.